lib/tokens.js

```diff
@@ -17,6 +16,9 @@ import pLimit from 'p-limit';
 import {promisify} from 'util';
 import Bitstring from '@digitalbazaar/bitstring';
 import {IdGenerator, IdEncoder} from 'bnid';
+import {constants, appContextMap, documentLoader} from 'cit-context';
+
+const {CIT_CONTEXT_URL} = constants;
```

Comment:
please destructure this directly out of `cit-context`, having a package specifics `constants` variable floating around seems confusing to me.  We want to set good patterns that others can follow.

Reply:
It's not possible to destructure directly out of an import (like it is with `const`). But I changed the import so that it's hopefully clearer.